tox.ini

```diff
@@ -139,8 +139,15 @@ deps =
 
 [pytest]
 # TODO: s/--strict/--strict-markers/ once xenial support is dropped
+testpaths = cloudinit tests/unittests
```

Comment:
I wonder if this allows us to drop our `posargs` defaulting from the other tox environments.